clang-tools-extra/docs/ReleaseNotes.rst

```diff
@@ -183,6 +183,10 @@ Changes in existing checks
   by adding the option `UseUpperCaseLiteralSuffix` to select the
   case of the literal suffix in fixes.
 
+- Improved :doc:`readability-redundant-casting
+  <clang-tidy/checks/readability/redundant-casting>` check to
+  exclude `CXXParenListInitExpr` for the source expression matcher.
```

Comment:
I don't think we should mention `CXXParenListInitExpr` as this is internal to clang-tidy and not something the end user is aware of.
What about ? 
```
- Improved :doc:`readability-redundant-casting
  <clang-tidy/checks/readability/redundant-casting>` check
  by addressing a false positive in aggregate initialization through 
  parenthesized list.
```

Reply:
Thanks for the suggestion.